web/src/beta/components/Icon/Icons/square.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect width="8" height="8" rx="1" fill="#C7C5C5"/>
```

Comment:
Please with any new icons added change the `fill` value to `currentColor` so that we can dynamically change the icon color.
If you are confused or unsure what I mean, please reference other svgs in `Icon/Icons/` 🙏🏻